core/ledger/kvledger/snapshot_mgmt.go

```diff
@@ -285,6 +287,7 @@ func newSnapshotRequestBookkeeper(dbHandle *leveldbhelper.DBHandle) (*snapshotRe
 
 // add adds the given block number to the bookkeeper db and returns an error if the block number already exists
 func (k *snapshotRequestBookkeeper) add(blockNumber uint64) error {
+	logger.Infow("Adding new request for snapshot", "blockNumber", blockNumber)
```

Comment:
Would it be difficult to add channelId to this log message (and also the next log messages)?